src/components/addBook.js

```diff
@@ -37,14 +38,27 @@ function AddBook() {
   };
 
   return (
-    <div className="form-inputs">
-      <h1>ADD A NEW BOOK</h1>
-      <form className="inputs" onSubmit={submit}>
+    <div className="Add-book">
+      <h1 className="form-heading">ADD A NEW BOOK</h1>
+      <form className="add-form" onSubmit={submit}>
         <input type="text" name="title" placeholder="Book-Title" value={state.title} onChange={read} />
         <input type="text" name="author" placeholder="Book-Author" value={state.author} onChange={read} />
+        <select
+          name="category"
+          key="book-category"
+          onChange={read}
+          value={state.category}
+        >
+          <option value="" hidden>Category</option>
+          <option value="Action">Action</option>
+          <option value="Fiction">Fiction</option>
+          <option value="Thriller">Thriller</option>
+          <option value="Inspiration">Inspiration</option>
+          <option value="Business">Business</option>
+        </select>
         <button type="submit">Add</button>
       </form>
```

Comment:
- [ ] Please add similar widths for the input fields and spaces between them as seen in the original zeplin design
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/201422901-ed4b03ed-6782-408f-99f4-88336cc2ee47.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/201422953-ca2ea9f1-812a-4cbf-8084-e48a8e7b7a76.png)